api/batch/resource.go

```diff
@@ -57,7 +57,7 @@ const (
 
 var (
 	// Hardcoded based on python/batch-predictor/docker/base.Dockerfile
-	mainApplicationPath = "local:///merlin-spark-app/main.py"
+	mainApplicationPath = "local:///home/spark/merlin-spark-app/main.py"
```

Comment:
I think this modification will not allow us to cleanly rollback. Maybe need to make it configurable as well.